plugins/include/amxmodx.inc

```diff
@@ -1826,6 +1836,7 @@ native set_user_flags(index, flags = -1, id = 0);
  *
  * @param index     Client index, 0 to set flags of server
  * @param id        Flag set id, ranging from 0 to 31
+ * @param override  If set to true, all other flags will be removed
```

Comment:
Misplaced param's description.

Reply:
Oops, I always get confused when viewing .inc files because of the way the description is placed above the native. Fixed.